### ISSUE TITLE
[NSFW] remove-class can't remove from html on Firefox

### ISSUE THREAD

**Issue by Yuki2718** [uBlockOrigin/uBlock-issues#1871]
### Prerequisites

- [X] I verified that this is not a filter issue (MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues))
- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue
- [X] The issue is not present after wholly disabling uBlock Origin ("uBO") in the browser
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] uBO with default lists/settings
- [X] using a new, unmodified browser profile

### Description

remove-class can't remove class from html, at least on `www.fuq.com` (https://github.com/AdguardTeam/AdguardFilters/issues/102675)

### A specific URL where the issue occurs

[NSFW] `https://www.fuq.com/ja/`

### Steps to Reproduce

1. Add `fuq.com##+js(rc, blurred, html)` possibly with any other last parameter such as stay
2. See `blurred` is not removed

### Expected behavior

blurred should be removed and on Chrome it's removed without specifying last parameter.

### Actual behavior

Not removed

### uBlock Origin version

1.39.2

### Browser name and version

95.0

### Operating System and version

Windows 10

**Comment by Yuki2718** [comment]
I guess I have read something similar about `html`, but can't find.

**Comment by gwarser** [comment]
Scriptlet blocked by CSP?

**Comment by Yuki2718** [comment]
Thx, missed that. https://github.com/AdguardTeam/AdguardFilters/issues/102675#issuecomment-991411672

**Comment by gwarser** [comment]
Duplicate of https://github.com/uBlockOrigin/uBlock-issues/issues/235
